src/ReactionMechanismSimulator.jl

```diff
@@ -1,83 +1,79 @@
-    const nasa = PyNULL()
-    const wilhoit = PyNULL()
-    const arrhenius = PyNULL()
-    const falloff = PyNULL()
-    const chebyshev = PyNULL()
-    const solvation = PyNULL()
-    const os = PyNULL()
-    function __init__()
-        copy!(Chem,pyimport_conda("rdkit.Chem","rdkit","rmg"))
-        copy!(Desc,pyimport_conda("rdkit.Chem.Descriptors","rdkit","rmg"))
-        try
-            copy!(molecule,pyimport("rmgpy.molecule"))
-            copy!(chemkin,pyimport("rmgpy.chemkin"))
-            copy!(species,pyimport("rmgpy.species"))
-            copy!(reaction,pyimport("rmgpy.reaction"))
-            copy!(nasa,pyimport("rmgpy.thermo.nasa"))
-            copy!(wilhoit,pyimport("rmgpy.thermo.wilhoit"))
-            copy!(arrhenius,pyimport("rmgpy.kinetics.arrhenius"))
-            copy!(falloff,pyimport("rmgpy.kinetics.falloff"))
-            copy!(chebyshev,pyimport("rmgpy.kinetics.chebyshev"))
-            copy!(solvation,pyimport("rmgpy.data.solvation"))
-            copy!(fragment,pyimport("rmgpy.molecule.fragment"))
-        catch e
-            copy!(molecule,pyimport_conda("molecule.molecule","rmgmolecule","hwpang"))
-            copy!(chemkin,pyimport_conda("molecule.chemkin","rmgmolecule","hwpang"))
-            copy!(species,pyimport_conda("molecule.species","rmgmolecule","hwpang"))
-            copy!(reaction,pyimport_conda("molecule.reaction","rmgmolecule","hwpang"))
-            copy!(nasa,pyimport_conda("molecule.thermo.nasa","rmgmolecule","hwpang"))
-            copy!(wilhoit,pyimport_conda("molecule.thermo.wilhoit","rmgmolecule","hwpang"))
-            copy!(arrhenius,pyimport_conda("molecule.kinetics.arrhenius","rmgmolecule","hwpang"))
-            copy!(falloff,pyimport_conda("molecule.kinetics.falloff","rmgmolecule","hwpang"))
-            copy!(chebyshev,pyimport_conda("molecule.kinetics.chebyshev","rmgmolecule","hwpang"))
-            copy!(solvation,pyimport_conda("molecule.data.solvation","rmgmolecule","hwpang"))
-            copy!(fragment,pyimport_conda("molecule.molecule.fragment","rmgmolecule","hwpang"))
-        end
-        copy!(pydot,pyimport_conda("pydot","pydot"))
-        copy!(os,pyimport_conda("os","os"))
+using PythonCall
+using CondaPkg
+packages = keys(CondaPkg.current_packages())
+const Chem = PythonCall.pynew()
+const Desc = PythonCall.pynew()
+const molecule = PythonCall.pynew()
+const chemkin = PythonCall.pynew()
+const species = PythonCall.pynew()
+const reaction = PythonCall.pynew()
+const nasa = PythonCall.pynew()
+const wilhoit = PythonCall.pynew()
+const arrhenius = PythonCall.pynew()
+const falloff = PythonCall.pynew()
+const chebyshev = PythonCall.pynew()
+const solvation = PythonCall.pynew()
+const fragment = PythonCall.pynew()
+const pydot = PythonCall.pynew()
+function __init__()
+    PythonCall.pycopy!(Chem, pyimport("rdkit.Chem"))
+    PythonCall.pycopy!(Desc, pyimport("rdkit.Chem.Descriptors"))
+    try
+        PythonCall.pycopy!(molecule, pyimport("rmgpy.molecule"))
```

Comment:
If this won't automatically install rmgmolecule do we need to make sure that happens in the build?

Reply:
What do you mean? Make the installation happen in the build? We currently install rmgmolecule in build

Reply:
Okay, this is fine. I was just clarifying we're now entirely dependent on installing rmgmolecule within the build.